.github/workflows/release_notes_comments_copy_from_pr_to_linked_issues.yml

```diff
@@ -0,0 +1,95 @@
+name: Release Notes Comments Copy from PR to Linked Issues
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [ master ]
+
+jobs:
+  copy_release_notes:
+    if: github.event.pull_request.merged == true
+    name: Copy Release Notes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch PR Comments
+        id: get-comments
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const prNumber = context.payload.pull_request.number;
+            const repoName = context.repo.repo;
+            const repoOwner = context.repo.owner;
+            const releaseNotesRegex = /release notes/i;
```

Comment:
this regex is different than the one above, I understand that both can be good enough, but perhaps let's be deterministic about this? =consistency

Reply:
Good catch!